complete/src/main/java/price/checker/domain/StockPrice.java

```diff
@@ -0,0 +1,125 @@
+package price.checker.domain;
+
+import javax.persistence.*;
+import java.text.DateFormat;
+import java.text.SimpleDateFormat;
+import java.util.Date;
+
+/**
+ * This is the model/schema for the StockPrice table stored in MySQL
+ * It uses an ID class called "StockPriceId"
+ * Because the StockPrice table uses a composite key, it's required for a
+ * separate, serializable class to be created for the key
+ */
+@Entity //This tells Hibernate to make a table out of this class
+@Table(name="Stock_Price")
+@IdClass(StockPriceId.class) //we need this and have the key in
+                            // another class because it is a composite key

```

Comment:
You should be able to use @Data here, which takes care of all the getters and setters, as well as the toString() function and the constructor.  https://projectlombok.org/features/Data

Check this class for reference:  https://github.com/galatea-associates/fuse-starter-java/blob/develop/src/main/java/org/galatea/starter/domain/SettlementMission.java

Reply:
Done -- but what if I want to specify the toString() output? Eg. What if I want the date object to be formatted a certain way when returned as a String?